pybop/optimisers/base_optimiser.py

```diff
@@ -405,3 +418,99 @@ def __str__(self) -> str:
             f"  Number of iterations: {self.n_iterations}\n"
             f"  SciPy result available: {'Yes' if self.scipy_result else 'No'}"
         )
+
+
+class MultiOptimisationResult:
+    """
+    Multi run optimisation result class. Stores the results
+    of multiple optimisation runs.
+
+    Attributes
```

Comment:
these are properties arn't they?

Reply:
also, `self.results` is not documented but can imagine would be useful to a user

Reply:
Nice catch - thanks!